visual-testing-app/src/App.jsx

```diff
@@ -0,0 +1,76 @@
+import { Component } from 'react';
+import { ThemeProvider } from '@emotion/react';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+import { customProperties } from '@commercetools-uikit/design-system';
+
+const darkTheme = {
+  colorSurface: 'black',
+  colorSolid: 'white',
+  colorNeutral60: 'rgba(255,255,255,0.60)',
+  colorNeutral: 'rgba(255,255,255,0.60)',
+  colorAccent98: 'rgba(0,0,0,0.98)',
+};
+const pkgComponentsModules = import.meta.globEager(
+  '../../packages/**/*.visualroute.jsx'
+);
```

Comment:
This is kind of the equivalent of Webpack's `require.context`

https://vitejs.dev/guide/features.html#glob-import